.travis.yml

```diff
@@ -19,7 +19,18 @@ script:
 - make vet
 - make website-test
 - make shellcheck
+- make website-githubpages
 
+deploy:
+  provider: pages
+  skip_cleanup: true
+  github_token: "$GITHUB_TOKEN"
+  target_branch: gh-pages
+  keep_history: true
+  local_dir: terraform-provider-website
+  on:
+    all_branches: true
```

Comment:
Could you change that so it runs only on the `master` branch now that you validated your test?